helm/templates/clusterrole.yaml

```diff
@@ -29,4 +29,32 @@ subjects:
 roleRef:
   kind: ClusterRole
   name: {{ include "vkaci.fullname" . }}-pods-list
+  apiGroup: rbac.authorization.k8s.io
+---

```

Comment:
ideally should be a separate file naming role.yaml which includes both role and rolebinding  as here in clusterrole.yaml